### PR TITLE
workshop operator create bundle

### PR DESCRIPTION
**Pull request by MaazMS**
Bundle created

### CODE REVIEW

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-2-24597f37`

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/stakater/workshop-operator
-  newTag: ocp4.6
+  newName: controller
```
Comment (rasheedamir) [inline config/manager/kustomization.yaml:7]: @MaazMS why name it controller? i think it should be
stakater/workshop-operator

Comment (MaazMS) [reply config/manager/kustomization.yaml:7]: It auto generated . I am fixing it .

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-2-03080d49`

**Review by rasheedamir — APPROVED** [review]

**Pull request merged**